docs/topics/api/reviews.rst

```diff
@@ -19,10 +19,12 @@ This endpoint allows you to fetch reviews for a given add-on.
 .. http:get:: /api/v3/addons/addon/(int:id|string:slug|string:guid)/reviews/
 
     :query string filter: The :ref:`filter <review-filtering-param>` to apply.
+    :query int show_grouped_ratings: Whether or not to show ratings aggregates for this add-on in the response.
     :>json int count: The number of results for this query.
     :>json string next: The URL of the next page of results.
     :>json string previous: The URL of the previous page of results.
     :>json array results: An array of :ref:`reviews <review-detail-object>`.
+    :>json object grouped_ratings: An object aggregating the number of times each rating has been posted for this add-on. Only present if ``show_grouped_ratings`` query parameter is present.
```

Comment:
For this case, I think an example of the object (in JSON) would clarify it. I forget if the ReST parameter markup allows you to add a code block in the parameter description, I'm pretty sure it does.